packages/database/create_env.ts

```diff
@@ -0,0 +1,78 @@
+import { execSync } from "node:child_process";
```

Comment:
I'm getting this error on multiple files in `package/database/`
<img width="1285" height="480" alt="image" src="https://github.com/user-attachments/assets/7e5d0ac2-7cf0-4183-ac39-7d69630acabe" />

Adding `project:true` to the `eslint.config` fixes it and matches other `eslint.config`s

```
languageOptions: {
      parserOptions: {
        project: true,
      },
    },
```

Reply:
Thanks for seeing this!

Reply:
We need to remove `project: "./tsconfig.json",` as well.